darwin/importer/importer.py

```diff
@@ -175,19 +177,26 @@ def import_annotations(
         If ``True`` appends the given annotations to the datasets. If ``False`` will override them.
     class_prompt : bool
         If ``False`` classes will be created and added to the datasets without requiring a user's prompt.
+    delete_for_empty : bool, default: False
+        If ``True`` will use empty annotation files to delete all annotations from the remote file.
+        If ``False``, empty annotation files will simply be skipped.
+        Only works for V2 datasets.
+        Takes precedence over the ``append`` flag.
```

Comment:
Let's please not do that. It's a very bad idea to allow a flag to override another one. Easy to loose data due to simple copy-paste mistake. Those flags should never be used together and darwin-py should IMO print error if someone tries.

Reply:
Fixed. Now I print an error and stop the process.